client/components/pages/admin/BlockUser.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <AdminCard
+    :title="isBlocked ? 'Unblock User' : 'Block User'"
+    :icon="isBlocked ? 'heroicons:lock-open-20-solid' : 'heroicons:no-symbol-20-solid'"
+  >
+    <div class="space-y-6 flex flex-col justify-between">
+      <p class="text-xs text-neutral-500">
+        <template v-if="isBlocked">
+          This will unblock the user and allow them to log in again. Their forms will remain in draft status.
+          <br>
+          <b>Blocked on:</b> {{ new Date(user.meta.blocked_at).toLocaleString() }}
+          <br>
+          <b>Reason:</b> {{ user.meta.block_reason }}
+        </template>
+        <template v-else>
+          This will block the user from accessing their account and set all their forms to draft.
+        </template>
+      </p>
+
+      <VForm @submit.prevent="submit">
+        <TextAreaInput
+          label="Reason"
```

Comment:
add help text to mention that user will be able to see reasons for both blokc and unblock